pkg/zson/zval.go

```diff
@@ -51,6 +51,9 @@ func ZvalToZeekString(typ zeek.Type, val []byte, isContainer bool) string {
 	var s string
 	switch typ.(type) {
 	case *zeek.TypeSet, *zeek.TypeVector:
+		if val == nil {
+			return "-"
+		}
```

Comment:
Instead of adding this, how about changing line 48 from `if val == nil && !isContainer {` to `if val == nil {`?

Reply:
yes

Reply:
yes